src/Compilers/Core/Portable/Operations/ControlFlowGraphExtensions.cs

```diff
@@ -15,7 +14,7 @@ public static partial class ControlFlowGraphExtensions
         /// Gets or creates a control flow graph for the given <paramref name="localFunction"/> defined in
         /// the given <paramref name="controlFlowGraph"/> or any of it's parent control flow graphs.
         /// </summary>
-        public static ControlFlowGraph GetLocalFunctionControlFlowGraphInScope(this ControlFlowGraph controlFlowGraph, IMethodSymbol localFunction, CancellationToken cancellationToken = default)
+        public static ControlFlowGraph GetLocalFunctionControlFlowGraphInScope([DisallowNull] this ControlFlowGraph? controlFlowGraph, IMethodSymbol localFunction, CancellationToken cancellationToken = default)
```

Comment:
❕ Considering this is a public API, requesting confirmation that this is a recommended practice

/cc @stephentoub

Reply:
I didn't notice this was public, thanks.